src/library/fixbook/FixBookUI.java

```diff
@@ -8,68 +8,90 @@
 
 	//public static enum uI_sTaTe { INITIALISED, READY, FIXING, COMPLETED };
-	public static enum uiState { INITIALISED, READY, FIXING, COMPLETED }; // changed enum name uI_sTaTe to uiState
+	public static enum UIState { INITIALISED, READY, FIXING, COMPLETED }; // changed enum name uI_sTaTe to UIState
 
-	private fIX_bOOK_cONTROL CoNtRoL;
-	private Scanner InPuT;
+	//private fIX_bOOK_cONTROL CoNtRoL;
+	private FixBookControl control; // changed variable type  fIX_bOOK_cONTROL to FixBookControl and CoNtRoL to control
+	//private Scanner InPuT;
+	private Scanner input; // changed variable name InPuT to input
 	//private uI_sTaTe StAtE;
-	private uiState state; // changed enum name uI_sTaTe to uiState
+	private UIState state; // changed enum name uI_sTaTe to UIState
 
 	
-	public FixBookUI(fIX_bOOK_cONTROL CoNtRoL) {
-		this.CoNtRoL = CoNtRoL;
-		InPuT = new Scanner(System.in);
+	public FixBookUI(FixBookControl control) {
+		//this.CoNtRoL = CoNtRoL;
+		this.control = control; //changed variable CoNtRoL to control
+		//InPuT = new Scanner(System.in);
+		input = new Scanner(System.in);
 		//StAtE = uI_sTaTe.INITIALISED;
-		state = uiState.INITIALISED; // changed enum name uI_sTaTe to uiState
-		CoNtRoL.SeT_Ui(this);
+		state = UIState.INITIALISED; // changed enum name uI_sTaTe to UIState
+		//control.SeT_Ui(this);
+		control.setUI(this);//changed variable CoNtRoL to control
+		
 	}
 
 
 	//public void SeT_StAtE(uI_sTaTe state) {
-	public void SeT_StAtE(uiState state) { // changed enum name uI_sTaTe to uiState
+	public void setState(UIState state) { // changed enum name uI_sTaTe to UIState and SeT_StAtE to setState
 		this.state = state;
 	}
 
 	
-	public void RuN() {
-		OuTpUt("Fix Book Use Case UI\n");
+	//public void RuN() {
+	public void run() { //change method name RuN to run
+		//OuTpUt("Fix Book Use Case UI\n");
+		output("Fix Book Use Case UI\n"); //changed method name OuTpUt to output
 		
 		while (true) {
 			
 			//switch (StAtE) {
 			switch (state) { // changed variable name stAtE to state
 			
 			case READY:
-				String BoOk_EnTrY_StRiNg = iNpUt("Scan Book (<enter> completes): ");
-				if (BoOk_EnTrY_StRiNg.length() == 0) 
-					CoNtRoL.SCannING_COMplete();
+				//String BoOk_EnTrY_StRiNg = iNpUt("Scan Book (<enter> completes): ");
+				String bookEntryString = input("Scan Book (<enter> completes): "); // changed variable name BoOk_EnTrY_StRiNg to bookEntryString
+				//if (BoOk_EnTrY_StRiNg.length() == 0) 
+				if (bookEntryString.length() == 0) {
+					//CoNtRoL.SCannING_COMplete();
+					control.SCannING_COMplete();
 				
-				else {
+				}else {
 					try {
-						int BoOk_Id = Integer.valueOf(BoOk_EnTrY_StRiNg).intValue();
-						CoNtRoL.BoOk_ScAnNeD(BoOk_Id);
+						//int BoOk_Id = Integer.valueOf(BoOk_EnTrY_StRiNg).intValue();
+						int bookId = Integer.valueOf(bookEntryString).intValue(); // changed variable name BoOk_EnTrY_StRiNg to bookEntryString
+						//CoNtRoL.BoOk_ScAnNeD(BoOk_Id);
+						control.BoOk_ScAnNeD(bookId);

```

Comment:
Hi Sankalpa, Method "BoOk_ScAnNeD" haven't been updated accordingly. Could you please have a look. Thank you!